packages/components/src/internal/components/entities/models.ts

```diff
@@ -136,23 +136,22 @@ export class EntityParentType extends Record({
             displayColumn = 'scientificName';
         }
 
-        return QueryColumn.create({
+        return new QueryColumn({
             caption: this.isAliquotParent ? QueryColumn.ALIQUOTED_FROM_CAPTION : formattedQueryName + captionSuffix,
             description: this.isAliquotParent
                 ? 'The parent sample of the aliquot'
                 : 'Contains optional parent entity for this ' + formattedQueryName,
             fieldKeyArray: [parentColName],
             fieldKey: parentColName,
-            lookup: {
+            lookup: new QueryLookup({
                 displayColumn,
                 isPublic: true,
                 keyColumn: 'RowId',
                 multiValued: this.isAliquotParent ? undefined : 'junction',
                 queryName: this.query,
                 schemaName: this.schema,
                 viewName: ViewInfo.DETAIL_NAME, // use the details view to assure we see values even if default view is filtered
-                table: this.isAliquotParent ? QueryColumn.MATERIAL_INPUTS : parentInputType,
```

Comment:
did @XingY confirm that this isn't needed? or did we determine where this came from or why it was there?

Reply:
yes Xing helped me confirm it is not necesssary